packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -466,31 +461,12 @@ export class GutenbergEditorPage {
 	 *
 	 * @returns {Promise<string>} Published article's URL.
 	 */
-	async getPublishedURL(): Promise< string > {
+	async getPublishedURLFromToast(): Promise< URL > {
 		const frame = await this.getEditorFrame();
 
-		const viewPublishedArticleButton = await frame.waitForSelector( selectors.viewButton );
-		return ( await viewPublishedArticleButton.getAttribute( 'href' ) ) as string;
-	}
-
-	/**
-	 * Closes the post-publish panel.
-	 */
-	async closePostPublishPanel(): Promise< void > {
-		const frame = await this.getEditorFrame();
-
-		await frame.click( selectors.closePublishPanel );
-	}
-
-	/**
-	 * Creates a new page/post using the post-publish panel.
-	 *
-	 * @param options will be forwarded to the button click action
-	 */
-	async postPublishAddNewItem( options: ClickOptions = {} ): Promise< void > {
-		const frame = await this.getEditorFrame();
-		await frame.waitForSelector( selectors.publishPanel );
-		await frame.click( selectors.addNewButton, options );
+		const toastLocator = frame.locator( selectors.toastViewPostLink );
+		const publishedURL = ( await toastLocator.getAttribute( 'href' ) ) as string;
+		return new URL( publishedURL );
```

Comment:
This method now only handles the obtaining of published article's URL from the toast.